src/api/index.js

```diff
@@ -133,42 +130,52 @@ class Api {
     return denormalize(this.state.favorites, [favoritesSchema], this.entities);
   }
 
+  getFavoritesTracks(n) {
+    return compose(
+      take(n || Infinity),
+      this.mapMetaProperty,
+      map(prop('track')),
+      this.getFavorites.bind(this),
+    )();
+  }
+
   /**
    * Add favorite
    * @param  {} track - track entity
    * @return {number[]} Favorites list
    */
-  addFavorite(track) {
+  addFavorite(track, n) {
```

Comment:
please add the JSDoc description for the **n** parameter

Reply:
done